src/deepsparse/engine.py

```diff
@@ -303,6 +305,13 @@ def cpu_vnni(self) -> bool:
         """
         return self._cpu_vnni
 
+    @property
+    def cpu_bf16(self) -> bool:
+        """
+        :return: True if bfloat16 support was detected on the cpu, False otherwise.
+        """
+        return self._cpu_bf16
+
```

Comment:
it would be nice if we could forward the properties from the `ARCH` class somehow instead of needing to duplicate  them each time